spec/lib/pg_search/multisearchable_spec.rb

```diff
@@ -69,8 +73,13 @@
         context "when the document is missing" do
           before { record.pg_search_document = nil }
 
-          describe "saving the record" do
+          describe "saving the record without changing any in it" do
+            it "should not create a PgSearch::Document record" do
+              expect { record.save! }.to change(PgSearch::Document, :count).by(0)
+            end
+
             it "should create a PgSearch::Document record" do
```

Comment:
Outer context no longer makes sense for this example, as it directly conflicts semantically with the previous sibling example.